GitCommands/RevisionReader.cs

```diff
@@ -416,7 +418,7 @@ DateTime ParseUnixDateTime()
             // NOTE the convention is that the Subject string is duplicated at the start of the Body string
             // Therefore we read the subject twice.
             // If there are not enough characters remaining for a body, then just assign the subject string directly.
-            var (body, additionalData) = ParseCommitBody(reader, subject);
+            var body = reader.ReadToEnd();
```

Comment:
This is rewritten in #9243 anyway.
```suggestion
            var body = reader.ReadToEnd()Trim();
```